circle.yml

```diff
@@ -1,10 +1,10 @@
 machine:
+  node:
+    version: 6.9.1
```

Comment:
I don't see that version in https://circleci.com/docs/build-image-trusty/#nodejs ?